org.openomf.OpenOMF.json

```diff
@@ -0,0 +1,190 @@
+{
+  "app-id": "org.openomf.OpenOMF",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openomf",
+  "permissions": [
+      "xdg-config",
+      "xdg-state"
+  ],
+  "finish-args": [
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "modules": [
+    {
+      "name": "libxmp",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libxmp/libxmp",
+          "tag": "libxmp-4.6.1"
+        }
+      ],
+      "config-opts": [
+          "-DBUILD_STATIC=ON",
+          "-DBUILD_SHARED=OFF"
+      ],
+      "cleanup": [
+        "/include",
+        "/share/doc",
+        "*.a",
+        "*.la"
+      ]
+    },
+    {
+      "name": "argtable3",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/argtable/argtable3",
+          "tag": "v3.2.2.f25c624"
+        }
+      ],
+      "config-opts": [
+          "-DARGTABLE3_ENABLE_EXAMPLES=OFF"
+      ],
+      "cleanup": [
+        "/include",
+        "/share/doc",
+        "*.a",
+        "*.la"
+      ]
+    },
+    {
+      "name": "enet",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://enet.bespin.org/download/enet-1.3.18.tar.gz",
+          "sha256": "2a8a0c5360d68bb4fcd11f2e4c47c69976e8d2c85b109dd7d60b1181a4f85d36"
+        }
+      ],
+      "cleanup": [
+        "/include",
+        "/share/doc",
+        "*.a",
+        "*.la"
+      ]
+    },
+    {
+      "name": "confuse",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libconfuse/libconfuse",
+          "tag": "v3.3"
+        }
+      ],
+    "make-install-args": "PREFIX=/app",
+      "cleanup": [
+        "/include",
+        "/share/doc",
+        "*.a",
+        "*.la"
+      ]
+    },
+    {
+      "name": "miniupnpc",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/transmission/miniupnpc",
+          "tag": "miniupnpc_2_3_0"
+        }
+      ],
+      "config-opts": [
+          "-DUPNPC_BUILD_STATIC=TRUE",
+          "-DUPNPC_BUILD_SHARED=FALSE",
```

Comment:
```suggestion
          "-DUPNPC_BUILD_STATIC=FALSE",
          "-DUPNPC_BUILD_SHARED=TRUE",
```

Reply:
use flatpak-builder 1.4.4 to the version that comes from flathub.

There is a bug in cmake that caused that regression.

We don't build static for flatpak.

Reply:
I'll have to figure out how to get that version locally and fix things, then.

Reply:
Installed the newer flatpak-builder, confirmed the cmake lib64 bug is gone, and enabled dynamic versions of all the dependencies.